ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/datepicker.html

```diff
@@ -36,6 +36,9 @@
             disabled="${!datePicker.enabled}"
             readonly="${datePicker.readOnly}"
             required="${datePicker.required}"
+            aria-label="${datePicker.label.visible && datePicker.label.value}"
```

Comment:
aria-label should not be required here (as `label for` is already there). any specific reason why this is added?

Reply:
removed.